config/nginx.sample.conf

```diff
@@ -47,7 +52,7 @@ server {
   gzip_vary on;
   gzip_min_length 1000;
   gzip_comp_level 5;
-  gzip_types application/json text/css text/javascript application/x-javascript application/javascript image/svg+xml application/wasm;
+  gzip_types application/json text/css text/javascript application/x-javascript application/javascript image/svg+xml application/wasm font/ttf;
```

Comment:
should we include other font types as well? or do those need no gzip compression?

Reply:
woff & woff2 are pre-compressed, so it's wasteful to gzip them as well.

But yeah I guess we can add `.otf`, just in case any theme/plugin is using it